lib/modules/backupAndRestore/providers/GoogleDrive.js

```diff
@@ -15,9 +15,9 @@ export class GoogleDrive extends Provider {
 
 	accessToken: string;
 
-	async init(): Promise<Provider> {
+	async init({ googleLoginHint }: *): Promise<Provider> {
 		this.accessToken = await launchAuthFlow({
-			domain: 'https://accounts.google.com/o/oauth2/v2/auth',
+			domain: `https://accounts.google.com/o/oauth2/v2/auth?login_hint=${googleLoginHint}`,
```

Comment:
FTR: I put this here because it's not a standard OAuth query param, and it seems like only google supports it (AFAICT)